<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CabMan Music</string>
  </property>
  <property name="windowIcon">
   <iconset resource="cabmanclient.qrc">
    <normaloff>:/icons/icons/music.png</normaloff>:/icons/icons/music.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-230</x>
      <y>640</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="song_name">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> </string>
    </property>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>75</y>
      <width>506</width>
      <height>185</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QSlider" name="progress_slider">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>43</y>
      <width>511</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>288</width>
      <height>33</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="previous_button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="cabmanclient.qrc">
           <normaloff>:/icons/icons/rewind.png</normaloff>:/icons/icons/rewind.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="play_button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="cabmanclient.qrc">
           <normaloff>:/icons/icons/play.png</normaloff>:/icons/icons/play.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pause_button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="cabmanclient.qrc">
           <normaloff>:/icons/icons/pause.png</normaloff>:/icons/icons/pause.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="next_button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="cabmanclient.qrc">
           <normaloff>:/icons/icons/forward.png</normaloff>:/icons/icons/forward.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="mute_button">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="cabmanclient.qrc">
         <normaloff>:/icons/icons/volume.png</normaloff>:/icons/icons/volume.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="volume_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>527</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="play_toolbar"/>
    <addaction name="pause_toolbar"/>
    <addaction name="repeat_toolbar"/>
    <addaction name="aleatory_toolbar"/>
    <addaction name="next_toolbar"/>
    <addaction name="previous_toolbar"/>
    <addaction name="separator"/>
    <addaction name="add_file_button"/>
    <addaction name="exit_toolbar"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="play_toolbar">
   <property name="icon">
    <iconset resource="cabmanclient.qrc">
     <normaloff>:/icons/icons/play.png</normaloff>:/icons/icons/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="pause_toolbar">
   <property name="icon">
    <iconset resource="cabmanclient.qrc">
     <normaloff>:/icons/icons/pause.png</normaloff>:/icons/icons/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="exit_toolbar">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="repeat_toolbar">
   <property name="icon">
    <iconset resource="cabmanclient.qrc">
     <normaloff>:/icons/icons/repeat.png</normaloff>:/icons/icons/repeat.png</iconset>
   </property>
   <property name="text">
    <string>Repeat</string>
   </property>
  </action>
  <action name="aleatory_toolbar">
   <property name="icon">
    <iconset resource="cabmanclient.qrc">
     <normaloff>:/icons/icons/aleatory.png</normaloff>:/icons/icons/aleatory.png</iconset>
   </property>
   <property name="text">
    <string>Aleatory</string>
   </property>
  </action>
  <action name="next_toolbar">
   <property name="icon">
    <iconset resource="cabmanclient.qrc">
     <normaloff>:/icons/icons/forward.png</normaloff>:/icons/icons/forward.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="previous_toolbar">
   <property name="icon">
    <iconset resource="cabmanclient.qrc">
     <normaloff>:/icons/icons/rewind.png</normaloff>:/icons/icons/rewind.png</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
  </action>
  <action name="show_playlist_button_toolbar">
   <property name="text">
    <string>Show Playlist</string>
   </property>
  </action>
  <action name="add_file_button">
   <property name="text">
    <string>Add file</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="cabmanclient.qrc"/>
 </resources>
 <connections/>
</ui>
